check_logs.py

```diff
@@ -258,6 +267,7 @@ def boot_test(build):
         show_builds()
         sys.exit(1)
     build = verify_build()
+    check_patch_failure(build)
```

Comment:
We already check `build["result"] == "fail"` in `verify_build`; let's move this check into `verify_build`. Otherwise this will still retry. If we already know that the the status message is due to patch application failure, then we should not retry.

Reply:
I am not sure I follow the "still retry" logic but sure, done. PTAL.